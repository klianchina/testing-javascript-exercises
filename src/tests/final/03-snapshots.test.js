import React from 'react';
import {render} from '@testing-library/react';
import Counter from '../../components/counter';

describe('Component: <Counter/>', () => {
  test('renders correctly', () => {
    const {container} = render(<Counter />);

    expect(container).toMatchSnapshot();
  });

  test('renders correctly', () => {
    const {container} = render(<Counter valuesToCount="tomatoes" />);
    const header = container.querySelector('h1')

    expect(header.textContent).toBe('Current tomatoes count: 0');
  });
})
