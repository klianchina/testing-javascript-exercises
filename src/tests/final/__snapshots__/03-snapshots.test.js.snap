// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: <Counter/> renders correctly 1`] = `
<div>
  <div>
    <h1>
      Current
       count: 
      0
    </h1>
    <button>
      Decrement
    </button>
    <button>
      Increment
    </button>
  </div>
</div>
`;
